dev/tidb-in-kubernetes/maintain/tidb-binlog.md

```diff
@@ -30,7 +30,105 @@ TiDB Binlog is disabled in the TiDB cluster by default. To create a TiDB cluster
         * MySQL compatible databases: enabled by setting `binlog.drainer.destDBType` to `mysql`. Meanwhile, you must configure the address and credential of the target database in `binlog.drainer.mysql`.
         * Apache Kafka: enabled by setting `binlog.drainer.destDBType` to `kafka`. Meanwhile, you must configure the zookeeper address and Kafka address of the target cluster in `binlog.drainer.kafka`.
 
-2. Create a new TiDB cluster or update an existing cluster:
+2. Set affinity and anti-affinity for TiDB and the Pump component:
+
+    > **Note:**
+    >
+    > If you enable TiDB Binlog in the production environment, it is recommended to set affinity and anti-affinity for TiDB and the Pump component; if you enable TiDB Binlog in a test environment on the internal network, you can skip this step.
+
+    By default, TiDB's affinity is set to `{}`. Currently, each TiDB instance does not have a corresponding Pump instance by default. When TiDB Binlog is enabled, if Pump and TiDB are separately deployed and network isolation occurs, and `ignore-error` is enabled, TiDB loses binlogs. In this situation, it is recommended to deploy a TiDB instance and a Pump instance on the same node using the affinity feature, and to split Pump instances on different nodes using the anti-affinity feature. For each node, only one Pump instance is required.
+
+    > Note:
+    >
+    > `<release-name>` needs to be replaced with the `Helm-release-name` of the target `tidb-cluster`.
```

Comment:
> `<release-name>` needs to be replaced with the `Helm-release-name` of the target `tidb-cluster`.

`<release-name>` needs to be replaced with the `Helm-release-name` of the target `tidb-cluster` chart.